src/components/SearchBar/SearchBar.vue

```diff
@@ -62,6 +62,7 @@
             v-for="result in searchResults"
             :key="result"
             class="text-gray-500 hover:text-primary-700"
+            @click="hide"
           >
             <slot :result="result" />
           </TableRow>
```

Comment:
we can also remove the commented-out import in l.75 but it won't let me suggest/comment on that line